NEWS.md

```diff
@@ -1,5 +1,10 @@
 # polars (development version)
 
+## Breaking changes and deprecations
+
+- `$apply()` on an Expr or a Series is renamed `$map_elements()`, and `$map()` 
+  is renamed `$map_batches()`. `$map()` and `$apply()` will be removed in 0.13.0 (#534).
```

Comment:
Wondering if this could be included in 0.11.1 and removed in 0.12.0.

Reply:
do you have other things you'd like to include in 0.11.1 apart from that? We release one version every 3-4 weeks so I think we'll get pretty quickly to 0.13.0

Reply:
I am ok to release 0.11.1 immediately after this is merged as I have nothing further in particular to add.

Reply:
My point was that I don't see the need to make a new release just for this deprecation. If you have other things to include then sure but otherwise let's just include this in 0.12.0?